### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Nodeist**
Add [Nodeist](https://github.com/Nodeist) resources.

### CODE REVIEW

**Review by ccamel — APPROVED** [review]
Great! Thank you! :+1:

File: README.md [hunk 1]
```diff
@@ -41,6 +42,8 @@
 | Whitepaper               | <https://docs.okp4.network/whitepaper/abstract>  | [OKP4](https://github.com/okp4) |
 | Nodes & Validator Guide  | <https://docs.okp4.network/nodes/introduction>   | [OKP4](https://github.com/okp4) |
 | Nodes & Validator Guide  | <https://nodes.guru/okp4/setup-guide/en>         | [Nodes Guru](https://nodes.guru/) |
+| Nodes & Validator Guide  | <https://nodeist.net/t/okp4/Setupen/>         | [Nodes Guru](https://nodes.guru/) |
+| Nodes & Validator Guide  | <https://nodes.guru/okp4/setup-guide/en>         | [Nodeist](https://nodeist.net/) |
```
Comment (bdeneux) [inline README.md:46]: It's seems that the url https://nodes.guru/okp4/setup-guide/en is written twice and the https://nodeist.net/t/okp4/Setupen/ url's is not in the good row. 

Here the correction if it's good : 
```suggestion
| Nodes & Validator Guide  | <https://nodes.guru/okp4/setup-guide/en>         | [Nodes Guru](https://nodes.guru/) |
| Nodes & Validator Guide  | <https://nodeist.net/t/okp4/Setupen/>         | [Nodeist](https://nodeist.net/) |
```

**Review by bdeneux — CHANGES_REQUESTED** [review]
I think there is a mistake in the table ;)

**Comment by Nodeist** [comment]
> I think there is a mistake in the table ;)

Thanks fixed!

**Comment by ccamel** [comment]
@Nodeist I just added a commit to resolve the conflict. It should be ok now. Thank you.

**Review by bdeneux — APPROVED** [review]
Nice, thank you 👍

**Review by hleonardtek — APPROVED** [review]

**Review by amimart — APPROVED** [review]
Nice thank you 😉

**Pull request merged**